config/a8c-for-agencies-horizon.json

```diff
@@ -36,7 +36,8 @@
 		"a4a-hosting-page-redesign": true,
 		"a4a-dev-sites": true,
 		"a4a-migrations-page-v2": true,
-		"a8c-for-agencies-agency-tier": true
+		"a8c-for-agencies-agency-tier": true,
+		"a4a-pressable-referrals": false
```

Comment:
Nitpicking: I believe setting this `true` in Horizon is safe.

Reply:
Yep, allowing it at Horizon is safe